node/src/components/consensus/protocols/highway/tests.rs

```diff
@@ -151,6 +151,68 @@ fn send_a_wire_unit_with_too_small_a_round_exp() {
     }
 }
 
+#[test]
+fn send_a_valid_wire_unit() {
+    let creator: ValidatorIndex = ValidatorIndex(0);
+    let state: State<ClContext> = new_test_state(&[100.into()], 0);
```

Comment:
Weights should also be an argument to `new_test_highway_protocol`. The latter has hardcoded stakes `vec![(*ALICE_PUBLIC_KEY, 100)]` and that's wrong. It makes it easy to have different stakes in "test state" and "test highway protocol".